tools/mo/openvino/tools/mo/utils/versions_checker.py

```diff
@@ -251,16 +251,14 @@ def check_requirements(framework=None):
     :param framework: framework name
     :return: exit code (0 - execution successful, 1 - error)
     """
+    framework_suffix = "_{}".format(framework)
     env_setup = get_environment_setup(framework)
     if framework is None:
         framework_suffix = ""
     elif framework == "tf":
-        if "tensorflow" in env_setup and env_setup["tensorflow"] >= LooseVersion("2.0.0"):
-            framework_suffix = "_tf2"
-        else:
-            framework_suffix = "_tf"
-    else:
-        framework_suffix = "_{}".format(framework)
+        if "tensorflow" in env_setup and env_setup["tensorflow"] < LooseVersion("2.0.0"):
+            log.error('\t\nTensorFlow1 usage detected. Since OpenVINO 2022.1 release it`s usage is deprecated.\n',
```

Comment:
Please write as follows:
```
Support of the Model Optimizer tool in TensorFlow 1.x environment is deprecated. It is highly recommended to use TensorFlow 2.x
```

Reply:
@asomsiko, please recommend the warning sentence if TensorFlow 1.x usage is detected